tools/run_tests/python_utils/jobset.py

```diff
@@ -357,7 +357,7 @@ def stdout(self=self):
                 if measure_cpu_costs:
                     m = re.search(
                         r'real\s+([0-9.]+)\nuser\s+([0-9.]+)\nsys\s+([0-9.]+)',
-                        stdout())
+                        strip_non_ascii_characters(stdout()).decode("ascii"))
```

Comment:
this is failing again. how about using `c.decode('utf8', errors='replace')`?  that should do exactly what we want.